charts/seed-bootstrap/charts/etcd-druid/templates/etcd-crd.yaml

```
@@ -0,0 +1,694 @@
+                kind:
+                  description: Kind of the referent
+                  type: string
+                name:
+                  description: Name of the referent
+                  type: string
+              type: object
+            labelSelector:
+              description: selector is a label query over pods that should match the
+                replica count. It must match the pod template's labels.
+              properties:
+                matchExpressions:
+                  description: matchExpressions is a list of label selector requirements.
+                    The requirements are ANDed.
+                  items:
+                    description: A label selector requirement is a selector that contains
+                      values, a key, and an operator that relates the key and values.
+                    properties:
+                      key:
+                        description: key is the label key that the selector applies
+                          to.
+                        type: string
+                      operator:
+                        description: operator represents a key's relationship to a
+                          set of values. Valid operators are In, NotIn, Exists and
+                          DoesNotExist.
+                        type: string
+                      values:
+                        description: values is an array of string values. If the operator
+                          is In or NotIn, the values array must be non-empty. If the
+                          operator is Exists or DoesNotExist, the values array must
+                          be empty. This array is replaced during a strategic merge
+                          patch.
+                        items:
+                          type: string
+                        type: array
+                    required:
+                    - key
+                    - operator
+                    type: object
+                  type: array
+                matchLabels:
+                  additionalProperties:
+                    type: string
+                  description: matchLabels is a map of {key,value} pairs. A single
+                    {key,value} in the matchLabels map is equivalent to an element
+                    of matchExpressions, whose key field is "key", the operator is
+                    "In", and the values array contains only "value". The requirements
+                    are ANDed.
+                  type: object
+              type: object
+            lastError:
+              type: string
+            ready:
+              type: boolean
+            readyReplicas:
+              format: int32
+              type: integer
+            replicas:
+              format: int32
+              type: integer
+            serviceName:
+              type: string
+            updatedReplicas:
+              format: int32
+              type: integer
+          type: object
+      type: object
```

Currently the minumum Seed Kubernetes version is v1.11. Currently it is not possible to apply this crd to v1.11 apiserver. You could check this suggestion.

```suggestion
      {{- if semverCompare ">= 1.12" .Capabilities.KubeVersion.GitVersion }}
      type: object
      {{- end }}
```